gax-httpjson/linkage-checker-exclusion.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
couldn't we simply not have such a file here?

Reply:
That would work. However the build.gradle would have if-statement to check whether the project is gax-grpc or not, to set the argument.

Reply:
Removed these empty files in favor of the if-statement in build.gradle.